test/vstest.console.UnitTests/Internal/ConsoleLoggerTests.cs

```diff
@@ -714,44 +758,6 @@ public void TestRunCompleteHandlerShouldWriteToConsoleIfTestsCanceledWithoutRunn
             this.mockOutput.Verify(o => o.WriteLine(CommandLineResources.TestRunCanceled, OutputLevel.Error), Times.Once());
         }
 
-        [TestMethod]
-        public void TestRunCompleteHandlerShouldNotWriteTolatTestToConsoleIfTestsCanceled()
-        {
-            var loggerEvents = new InternalTestLoggerEvents(TestSessionMessageLogger.Instance);
-            loggerEvents.EnableEvents();
-            var parameters = new Dictionary<string, string>();
-            parameters.Add("verbosity", "normal");
-            this.consoleLogger.Initialize(loggerEvents, parameters);
-
-            foreach (var testResult in this.GetTestResultObject(TestOutcome.Failed))
-            {
-                loggerEvents.RaiseTestResult(new TestResultEventArgs(testResult));
-            }
-            loggerEvents.CompleteTestRun(null, true, false, null, null, new TimeSpan(1, 0, 0, 0));
-
-            this.mockOutput.Verify(o => o.WriteLine(string.Format(CultureInfo.CurrentCulture, CommandLineResources.TestRunSummaryForCanceledOrAbortedRun, 0, 1, 0), OutputLevel.Information), Times.Once());
-            this.mockOutput.Verify(o => o.WriteLine(CommandLineResources.TestRunCanceled, OutputLevel.Error), Times.Once());
-        }
-
-        [TestMethod]
-        public void TestRunCompleteHandlerShouldNotWriteTolatTestToConsoleIfTestsAborted()
```

Comment:
>TestRunCompleteHandlerShouldNotWriteTolatTestToConsoleIfTestsAborted [](start = 20, length = 68)

Why have we removed these two tests? Can you please tell what will be the new behavior in case of these scenario?